interface/src/project/validators.ts

```diff
@@ -412,13 +424,30 @@ export const isGPIOUniqueValidator = (sensors: AnalogSensor[]) => ({
   }
 });
 
+export const uniqueAnalogNameValidator = (sensors: AnalogSensor[]) => ({
+  validator(rule: InternalRuleItem, n: string, callback: (error?: string) => void) {
+    if (n !== '' && sensors.find((as) => as.n === n)) {
```

Comment:
should we check for not being case-sensitive? compare lower() to lower() ?

Reply:
Yes, i add a commit.